onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -1040,12 +1045,14 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
         is_distill = False
         is_distill_add = False
         is_no_mask_attention = False
+        has_rel_pos_bias = False
         qk_paths = {
             "path1": (["Softmax", "Add", "Div", "MatMul"], [0, 0, None, 0]),
             "path2": (["Softmax", "Add", "Mul", "MatMul"], [0, 0, None, 0]),
             "path3": (["Softmax", "Where", "MatMul", "Div"], [0, 0, 2, 0]),
             "path4": (["Softmax", "Add", "Where", "MatMul"], [0, 0, 0, 2]),
             "path5": (["Softmax", "Div", "MatMul"], [0, 0, 0]),
+            "path6": (["Softmax", "Add", "Add", "Div", "MatMul"], [0, 0, None, 0, 0]),
```

Comment:
This pattern is different from the image in description:
![image](https://github.com/microsoft/onnxruntime/assets/30328909/f55e3762-b22b-41e5-9a17-4d2ea6a0846b)

Reply:
Yes